### PR TITLE
CRDCDH-1113 UAT: Section B updates and resizing multiline inputs

### PR DESCRIPTION
**Pull request by Alejandro-Vega**
### Overview

Increased Section B Program Title character limit to 100, and updated Program Description and Study Description to be resizable inputs. 

### Change Details (Specifics)

- Updated Program title character limit from 50 => 100
- Updated Program Description and Study Description to be resizable inputs
- Updated Text input component to allow infinite vertical resizing
- Set a minHeight when resizing is available so that it cannot be resized smaller than the original, this required some overwriting of MUI styling

### Related Ticket(s)

[CRDCDH-1113](https://tracker.nci.nih.gov/browse/CRDCDH-1113)

### CODE REVIEW

File: src/components/Questionnaire/TextInput.tsx [hunk 1]
```diff
@@ -70,8 +73,13 @@ const StyledOutlinedInput = styled(OutlinedInput)(() => ({
   "&.MuiInputBase-multiline": {
     padding: "12px",
   },
+  "& .MuiInputBase-inputMultiline": {
```
Comment (amattu2) [inline src/components/Questionnaire/TextInput.tsx:76]: If you specify `multiline` but don't provide `rows`, we get styling with `NaNpx` values. Whether you want to address this or not is your call, but I just wanted to point it out.

Screenshot 2024-05-13 at 2 49 33 PM

File: src/components/Questionnaire/TextInput.tsx [hunk 2]
```diff
@@ -55,7 +55,10 @@ const StyledHelperText = styled(FormHelperText)(() => ({
   minHeight: "20px",
 }));
 
-const StyledOutlinedInput = styled(OutlinedInput)(() => ({
+const StyledOutlinedInput = styled(OutlinedInput, {
+  shouldForwardProp: (prop) => prop !== "resize" && prop !== "lineHeight",
```
[the remark this answers is not in the document]
Comment (Alejandro-Vega) [reply src/components/Questionnaire/TextInput.tsx]: Oops, forgot to change! Thanks!

File: src/components/Questionnaire/TextInput.tsx (hunk 1 above)
Comment (Alejandro-Vega) [reply src/components/Questionnaire/TextInput.tsx]: Updated to default to 1 * row height instead.

**Review by amattu2 — APPROVED** [review]
LGTM. Did not retest the changes.

**Pull request merged**